pkg/sriov/sriov.go

```diff
@@ -288,7 +308,11 @@ func (s *sriovManager) ResetVFConfig(conf *types.NetConf) error {
 			conf.HostIFGUID = "FF:FF:FF:FF:FF:FF:FF:FF"
 		}
 
-		return s.setVfGUID(conf, pfLink, conf.HostIFGUID)
+		if err := s.setVfGUID(conf, pfLink, conf.HostIFGUID); err != nil {
+			return err
+		}
+		// setVfGUID cause VF to rebind, which change its name. Lets restore it.
```

Comment:
can you add a comment saying that once setVfGUID does not rebind VF to apply the GUID change this should be removed ?